tests/improver-blend-adjacent-points/02_basic_mean.bats

```diff
@@ -36,8 +36,8 @@
   KGO="blend_adjacent_points/basic_mean/kgo.nc"
 
   # Run weighted blending with weighted_mean mode and check it passes.
-  run improver blend-adjacent-points 'forecast_period' 'weighted_mean' 3.0 \
-      --parameter_unit 'hours' \
+  run improver blend-adjacent-points 'forecast_period' '2' --parameter_unit 'hours' \
+      3.0 'weighted_mean' \
       "$IMPROVER_ACC_TEST_DIR/blend_adjacent_points/basic_mean/multiple_probabilities_rain_*H.nc" \
```

Comment:
While not caused by this PR, can the test data directory be renamed to match the test script? (replace _ with -)

Reply:
Done.